doc/source/whatsnew/v0.20.0.txt

```diff
@@ -389,7 +389,7 @@ Bug Fixes
 
 - Bug in compat for passing long integers to ``Timestamp.replace`` (:issue:`15030`)
 - Bug in ``.loc`` that would not return the correct dtype for scalar access for a DataFrame (:issue:`11617`)
-
+- Bug in ``CategoricalIndex``, a missing method is causing ``groupby`` to fail (:issue:`15155`)
```

Comment:
Can you re-phrase this as "Bug in ``GroupBy.get_group`` failing with a categorical grouper (:issue`15155`)"?